content/300-guides/400-migrate-to-prisma/01-migrate-from-typeorm.mdx

```diff
@@ -398,7 +398,7 @@ Assume you have the following database connection details in `ormconfig.json`:
 
 ```json file=ormconfig.json
 {
-  "type": "mssql",
+  "type": "sql-server",
```

Comment:
I am not sure if this will work in TypeORM.

Reply:
True, I just checked, that's `mssql`